src/lib/createStardustComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import createComponentInternal from './createComponent'
+import * as React from 'react'
+import { pick } from 'lodash'
+import { ComponentSlotClasses } from '@stardust-ui/react'
```

Comment:
This thing should be imported from `/lib` or `/component` I think.

Reply:
Changed, imported from themes/types